Gemfile

```diff
@@ -42,6 +42,7 @@ gem 'twilio-ruby', '~> 5.20.1'
 gem 'sidekiq'
 gem 'newrelic_rpm'
 gem 'capybara-email'
+gem 'table_print'
```

Comment:
what is this magical gem?

Reply:
Per Ben - this helps print ActiveRecord queries in a table format

Reply:
🎉